packages/extension/src/view/devtools/pages/privateAdvertising/protectedAudience/prebid.tsx

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { Resizable } from 're-resizable';
+import { JsonView } from '@google-psat/design-system';
+/**
+ * Internal dependencies
+ */
+import { useProtectedAudience } from '../../../stateProviders';
+
+const PrebidPanel = () => {
+  const { functionKeys, getPrebidData, selectedProperty, prebidResponse } =
+    useProtectedAudience(({ state, actions }) => ({
```

Comment:
Possible to keep Prebid separate from PA?

Reply:
Would like to take this in a follow-up PR since I am not sure how many changes this might bring in this PR if that works.

Reply:
Such separation is needed.